css/styles.css

```diff
@@ -4,6 +4,52 @@
   padding: 0;
 }
 
+/* #myModal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 27%;
+  left: 0%;
+  top: 0%;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(39, 4, 197, 0.4);
+}
+
+.modal-content {
+  display: flex;
+  background-color: #fefefe;
+  justify-content: space-around;
```

Comment:
- [ ] Kindly do well to remove comented code from your codebase. This is to make sure you don't have a clustered project and codease 👍